openedx/core/djangoapps/persistenttask/models.py

```diff
@@ -0,0 +1,31 @@
+"""
+Models to support persistent tasks.
+"""
+
+from django.db import models
+
+from openedx.core.lib.properties import JSONProperty
+
+
+class FailedTask(models.Model):
+    """
+    Representation of tasks that have failed
+    """
+    task_name = models.CharField(max_length=255, db_index=True)
+    task_id = models.CharField(max_length=255)
+    argstring = models.TextField(blank=True)
+    kwargstring = models.TextField(blank=True)
+    exc = models.CharField(max_length=255)
```

Comment:
Do you want to index exc?  Or include it in any indexes?
Thinking about access patterns, I assume the management command is going to do something like
select * from persistenttask_failedtasks where exc = foo order by failed?
or is it just select * from persistenttask_failedtasks where resolved is NULL order by failed?